### ISSUE TITLE
Rendre les diamants noble et émeraudes polies VRAIMENT utiles

### ISSUE THREAD

**Issue by cadenza-k** [HollishKid/Gaia#126]
Je m'explique:

A l'heure actuelle, tout le monde se balade en stuff vaillant, ou fer volcanique car peu cher et très intéressant, or, les pièces d'équipement Noble sont très cher à réaliser (diams noble + emeraude polie) et nécessitent autant de niveau que pour réaliser du stuff diamant volca.

Donc, avec @Axeloow nous avons envisagé de rendre nos chers confrères d'Ingor un peu plus envieux envers cette ressource: 

- Les nains possèdent le volcan qui est leur unique centre de minage, or les minerais extrait de cette zone sont incompatible avec ces crafts-là, donc, pourquoi pas intégrer un pnj à l'entrée la mine (juste après le check bq d'entrée) qui achète l'équipement noble, et ce, à un prix plutôt avantageux pour le joueur (mais pas trop, sinon, il va devenir riche en trop peu de temps)

 - Evidemment, cette avantage matériel ne serait exploitable uniquement avec un niveau de forge assez élevé pour empêcher les joueurs les moins expérimentés d'y accéder (ex rendre le niveau de fonte requis encore plus élevé)

(petite note: faudra penser aussi à reequilibrer les prixs de vente à l'admin shop 8* la vente d'un diamant, ça peux être drôle pour nous mais pas forcément pour le joueur qui vends ses objets pour se racheter une future épée vaillante ;) )

**Comment by Undertimes** [comment]
J'ai fait une petite proposition il y a longtemps, simple à mettre en place et permettant de rendre les minerais raffinés utiles. Il faudrait changer les armures et armes associées pour qu'elle aient une utilité particulière, en comparaison du diamant volcanique.

Voila mon idée : 
- Donner de meilleures stats défensives au diamant volcanique, en faire une vraie armure de tank. Peut-être faire que cette armure ralentisse un petit peu son porteur.
- Garder des stats d'armures assez faibles pour le diams noble (au niveau de l'armure vaillante) mais lui donner des bonus de vitesse, entre 5 et 10% de vitesse en plus par pièce d'armure équipée.

Ainsi, le joueur pourrait choisir entre 3 styles de jeu : 
- Armure vaillante : moins puissant, mais équilibré et incassable.
- Armure volcanique : lent et résistant.
- Armure noble : léger et rapide.

**Comment by cadenza-k** [comment]
Dans ce cas là, il y aurait possibilité de faire un mix volca/noble pour une armure équilibrée un peu plus OP mais cassable 🤔

**Comment by Undertimes** [comment]
Yup. De cette façon, les joueurs pourraient créer leur propre combinaison d'armures, en fonction de leurs goûts. Ils pourraient jouer sur ces 3 types d'armures, et trouver le style qui leur convient le mieux, au lieu de juste prendre un set complet plus puissant que tout le reste.

**Comment by cadenza-k** [comment]
Mais y'en aura qui prendrons toujours le set le plus op sans réfléchir xD

**Comment by Undertimes** [comment]
L'intérêt serai de ne pas avoir de set "OP" mais plutôt d'avoir des sets égaux en puissance, mais avec des forces et des faiblesses variées.
